apps/api/prisma/seed-samples.js

```diff
@@ -668,3 +668,50 @@ export const caseStatusNames = [
 	{ name: 'post_decision' },
 	{ name: 'withdrawn' }
 ];
+
+export const folders = [
+	{
+		displayNameEn: 'Project management',
+		displayOrder: 100
+	},
+	{
+		displayNameEn: 'Legal advice',
+		displayOrder: 200
+	},
+	{
+		displayNameEn: 'Transboundary',
+		displayOrder: 300
+	},
+	{
+		displayNameEn: 'Legal rights',
+		displayOrder: 400
+	},
+	{
+		displayNameEn: 'S51 advice',
+		displayOrder: 500
+	},
+	{
+		displayNameEn: 'Pre-application',
+		displayOrder: 600
+	},
+	{
+		displayNameEn: 'Acceptance',
+		displayOrder: 700
+	},
+	{
+		displayNameEn: 'Relevant representation',
```

Comment:
representations is pluralised in requirements.  I note that the Figma has mis-spelt representations though as "reperesentations" (which should be ignored!)